### PR TITLE
📦 NEW: Dockerfile

### PR DESCRIPTION
**Pull request by sbkg0002**
Add simple Dockerfile to use as a Linux container.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -34,6 +34,9 @@ npm install -g corona-cli
 
 # Or run directly with npx (installs CLI on every run).
 npx corona-cli
+
+# Or run as a Docker container
+docker run -it --rm sbkg0002/corona-cli:latest
```
Comment (ahmadawais) [inline readme.md]: What is `sbkg0002/corona-cli`

**Comment by sbkg0002** [comment]
Just my namespace on dockerhub, that contains the image. I can remove it if needed.

**Comment by ahmadawais** [comment]
Please do remove this. I think having a docker file is good enough, people
with the need to run the CLI that way would just use it.

>

**Comment by sbkg0002** [comment]
Reverted the readme.

**Review by ahmadawais — APPROVED** [review]
Looks good to me! ✔︎

**Comment by ahmadawais** [comment]
Thank you! 👌

**Pull request merged**